.circleci/config.yml

```diff
@@ -538,6 +538,11 @@ workflows:
           py_version: "3.8"
           context: e2e-testing
           filters: *filter_semver_tags
+      - container_tests:
```

Comment:
I think we should probably just pull `containter_tests` out of the prod workflow and stick it into the RC workflow as a gate to the 'push_image' job. The prod image workflow just retags the last RC image, so this will provide us the tests we need early on in the release pipeline.

Reply:
sounds good. i've removed test-container-prod. Thanks